csrankings.csv

```diff
@@ -6753,6 +6759,7 @@ Liam Paull,University of Montreal,http://people.csail.mit.edu/lpaull/,H9xADK0AAA
 Liam Peyton,University of Ottawa,http://www.site.uottawa.ca/~lpeyton/,jKBUKKgAAAAJ
 Liam Roditty,Bar-Ilan University,https://u.cs.biu.ac.il/~liamr/,UY5xtq0AAAAJ
 Liang Huang 0001,Oregon State University,http://eecs.oregonstate.edu/~huanlian/,AdFeG5cAAAAJ
+Liang Wang 0001,Chinese Academy of Sciences,http://www.cbsr.ia.ac.cn/users/liangwang/,8kzzUboAAAAJ

```

Comment:
Is the Institute for Automation part of a CS department? Does it grant degrees in CS?

Reply:
Dear Prof. Berger,

I have corrected the homepages of the authors. I just found they all have the ucas.ac.cn homepage with the format as "http://people.ucas.ac.cn/~"+AUTHORNAME.

In Chinese Academy of Sciences, CS guys are in the school of computer and control engineering (http://scce.ucas.ac.cn/index.php/zh-CN/szdw/gwjs). I am very sorry for the ugly English website. But it contains full information in the Chinese version including all these professors. I am pretty sure that these professors advise computer science PhD students from the school of computer and control engineering because computer science is separated into computer application technology, software, computer systems mainly in China for master and PhD degree and these professors' majors are consistent with these sub-majors from the above website. 

Thank you very much for your great work!

Wentao

Reply:
Dear Prof. Berger,

Like many institutes or centers in US, these institutes cannot grant degrees. All the degrees or students are all from University of Chinese Academy of Sciences. So the professors in these institutes are master or PhD advisers in University of Chinese Academy of Sciences. The institutes or centers are research units. The University or school is the educational affiliation.

Thank you!

Wentao